File: exercises-cpp/1. Secuenciales/13.cpp
#include <iostream>
#include <cstdio>
#include <cmath>
using namespace std;

int main()
{
    system("cls");
    
    float a,b,hipotenusa;
    printf("Cateto adyasente: "); cin>>a;
    printf("Cateto opuesto: "); cin>>b;

    hipotenusa = sqrt((a * a) + (b * b));

    cout<<"Hipottenusa: "<<hipotenusa<<endl;
}

File: exercises-python/1. Problemas secuenciales/13.py
'''
Desarrolle el programa que permita calcular la hipotenusa de un triángulo. Usando la clase Math
'''
import os
os.system("cls")
import math

#Datos
c1 = int(input("Cateto adyasente: "))
c2 = int(input("Cateto opuesto: "))

#Proceso

hipotenusa = math.sqrt((c1 ** 2) + (c2 ** 2))

#Salida
print(f"Hipottenusa: {hipotenusa}")